app/components/Sidebar/components/CollectionLink.js

```diff
@@ -74,49 +78,103 @@ function CollectionLink({
     }),
   });
 
+  //Drop to reorder Collection
+  const [{ isCollectionDropping }, dropToReorderCollection] = useDrop({
+    accept: "collection",
+    drop: async (item, monitor) => {
+      collections.move(
+        item.id,
+        fractionalIndex(collection.index, belowCollectionIndex)
+      );
+    },
+    canDrop: (item, monitor) => {
+      return (
+        collection.id !== item.id &&
+        (!belowCollection || item.id !== belowCollection.id)
+      );
+    },
+    collect: (monitor) => ({
+      isCollectionDropping: monitor.isOver(),
+    }),
+  });
+
+  // Drag to reorder Collection
+  const [{ isCollectionDragging }, dragToReorderCollection] = useDrag({
+    item: {
+      type: "collection",
+      activeCollectionId: ui.activeCollectionId,
+      id: collection.id,
+    },
+    collect: (monitor) => ({
+      isCollectionDragging: monitor.isDragging(),
+    }),
+    canDrag: (monitor) => {
+      return can.move;
+    },
+    begin: (monitor) => {
+      ui.activeCollectionId = "";
+    },
```

Comment:
```suggestion
    begin: (monitor) => {
      setTimeout(() => {
        ui.clearActiveCollection();
      }, 0);
    },
```

In my testing this needs to be moved to next frame otherwise collections below the active collection are not draggable, this seems like it _might_ be a browser quirk

Reply:
Indeed, it was a bug. I should have checked it. Thanks :smile: